pkg/deploy/generator/resources_gateway.go

```diff
@@ -644,6 +644,9 @@ for scan in baseline clamav software; do
   /usr/local/bin/azsecd config -s $scan -d P1D
 done
 
+# We need to manually set PasswordAuthenticatin to true in order for the VMSS Access JIT to work
+sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config
```

Comment:
is `PasswordAuthentication no` always in `/etc/ssh/sshd_config`?

Reply:
yes, it actually appears twice. Once commented out and once uncommented and always with the "no" value.